modules/dnn/misc/java/test/DnnTensorFlowTest.java

```diff
@@ -0,0 +1,103 @@
+package org.opencv.test.dnn;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import org.opencv.core.Core;
+import org.opencv.core.Mat;
+import org.opencv.core.Scalar;
+import org.opencv.core.Size;
+import org.opencv.dnn.DictValue;
+import org.opencv.dnn.Dnn;
+import org.opencv.dnn.Importer;
+import org.opencv.dnn.Layer;
+import org.opencv.dnn.Net;
+import org.opencv.imgcodecs.Imgcodecs;
+import org.opencv.imgproc.Imgproc;
+import org.opencv.test.OpenCVTestCase;
+
+public class DnnTensorFlowTest extends OpenCVTestCase {
+
+    String modelFileName = "";
+    String sourceImageFile = "";
+
+    Net net;
+
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+
+        String path = System.getenv("OPENCV_DNN_TEST_DATA_PATH");
+
+        if(path == null){
+            isTestCaseEnabled = false;
+            return;
+        }
+
+        File currentRelativePath = new File(System.getenv("OPENCV_DNN_TEST_DATA_PATH"));
+        modelFileName =  new File(currentRelativePath, "dnn/tensorflow_inception_graph.pb").toString();
+
+        File f = new File(currentRelativePath, "dnn/space_shuttle.jpg");
```

Comment:
opencv_extra testdata files are located under
`OPENCV_TEST_DATA_PATH`
env variable instead of
`OPENCV_DNN_TEST_DATA_PATH`